packages/test-utils/define-getter.ts

```diff
@@ -0,0 +1,8 @@
+import { isFunction } from 'lodash'
+
+const defineGetter = (obj: Record<string, unknown>, prop: string, value: any) => {
+  const getter = isFunction(value) ? value : () => value
+  Object.defineProperty(obj, prop, { get: getter })
+}
+
+export default defineGetter
```

Comment:
When I tested the `Slider` component, I extracted a test util function of `jest.spyOn`, and the returned object of it provides a method of restoring. I think use it is better.

File path: `utils/jestMock.ts`, but now I want move it to `test-utils` in #250 .

Reply:
I do not think this has anything to do with your PR, I'm excluding you out of this PR.